lava-provider/example_rpcprovider.yml

```diff
@@ -0,0 +1,27 @@
+endpoints:
+  - api-interface: tendermintrpc
+    chain-id: LAV1
+    network-address:
+        address: "0.0.0.0:2001"
+    node-urls:
+      - url: http://provider.hurricane.akash.pub:<26657>
```

Comment:
since that's an example, let's remove the `<` & `>` bits as this may confuse some folks.